bip-0011.mediawiki

```diff
@@ -54,7 +54,7 @@ A weaker argument is OP_CHECKMULTISIG should not be used because it pops one too
 
 OP_CHECKMULTISIG is already supported by old clients and miners as a non-standard transaction type.
 
-https://github.com/gavinandresen/bitcoin-git/tree/op_eval
+https://github.com/gavinandresen/bitcoin-git/tree/e679ec969c8b22c676ebb10bea1038f6c8f13b33
```

Comment:
This is an old commit. Why not the most recent 77f21f1583deb89bf3fffe80fe9b181fedb1dd60?

Or perhaps link https://github.com/bitcoin/bitcoin/pull/669/files ?

Reply:
### Updated Commit 

Ack'd to fixing  with to the more modern commit.

The reason I chose the commit was it was referenced as the OP_EVAL implementation by Gavin Andresen in bip11/12 and would match the branch at the time the BIP was submitted.  Though yours is newer and would represent the final state of the branch before the branch was removed. 

My method to determine the location of the deleted branch was to run 
```git log --all --full-history -- "**op_eval*"```

The result provided this as well as your commit linked.

![image](https://user-images.githubusercontent.com/3104223/193160785-9ba4261f-c743-438e-b18d-a5cdbe97bfa2.png)


### In regards to the "Or perhaps".
I feel this would be the most appropriate solution for the brokens links as it was the entire merged implementation of the code changes, but isn't Gavin Andresen original 'implementation'.

### Advantages to this method

- Actual implementation into BitcoinQt is clear.
- Preserved as a part of the bitcoin/bitcoin repo.

### Disadvantages to this method

- Doesn't wholly represent the BIP authors implementation. 


### Closing statement.
I would like to implement it as a link to Gavins implementation as to preserve the original BIP as much as possible.

If you would like to use the link to the OP_EVAL and merge into bitcoin/bitcoin I can do that as well. Please let me know which fix you would like and I will implement immediately.

Reply:
I'm not sure there was a distinction between the "original" and the final version in this sense - BIPs typically get updated as the proposal changes.

But ultimately, it's a question of what @gavinandresen will ACK here.